package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "appoptics-apm",
-  "version": "8.0.1",
+  "version": "8.1.0",
```

Comment:
why not 8.0.2? 
I seems like a fix to me

Reply:
i bumped the minor version because i didn't just fix the bug but also added the feature to allow mapping a different header to the ClientIP KV. that's likely to be useful in nginx or other reverse-proxy environments.